hs628_tools/Outlier_debugged.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
+import csv
+import matplotlib.pyplot as plt
+
+##Class Outliers:
```

Comment:
You may need to uncomment this line to initiate your class Outliers.

Reply:
Thanks for commenting Omid: I made the change for the work. :)